tools/c7n_kube/c7n_kube/server.py

```diff
@@ -0,0 +1,145 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+import json
+import os
+import http.server
+
+from c7n.config import Config
+from c7n.loader import DirectoryLoader
+
+import logging
+
+log = logging.getLogger("c7n_kube.server")
+log.setLevel(logging.DEBUG)
+
+
+HOST = "0.0.0.0"
+
+
+class AdmissionControllerServer(http.server.HTTPServer):
```

Comment:
http server is effectively single threaded and low level, which doesn't nesc feel like a great solution for real world k8s deployments vs an async framework, but happy to defer this to a followup.